modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -493,6 +493,34 @@ $ docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
 CLEAN_PERFORMANCE_PROFILE="false" registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
 ----
 
+[id="cnf-performing-end-to-end-tests-running-in-single-node-cluster_{context}"]
+== Running in single node cluster
+
+Running tests on a single node cluster causes the following limitations to be imposed:
+
+* Longer timeouts for certain tests, including SR-IOV and SCTP tests
+* Tests requiring master/worker nodes are skipped
```

Comment:
* Suggest "or" instead of the slash ("Tests requiring master or worker nodes are skipped")
* Can you replace "master" with "control plane" and have this hold the same meaning?